automated_analysis.py

```diff
@@ -275,6 +301,17 @@ def update_survey_counts(survey_counts, td):
                 themes["Total Relevant Participants"]["Total Participants"] += 1
                 update_survey_counts(themes["Total Relevant Participants"], td)
 
+            set_survey_percentages(themes["Total Relevant Participants"], themes["Total Relevant Participants"])
+
+            for cc in episode_plan.coding_configurations:
+                assert cc.coding_mode == CodingModes.MULTIPLE
```

Comment:
Consider adding the same error message as above (`"Other CodingModes not (yet) supported"`)?

Reply:
updated, thanks